db/forms/kpi/gbv_assessment.rb

```diff
@@ -0,0 +1,28 @@
+gbv_assessment_fields = [
+  Field.new({"name" => "completed_asssessment",
```

Comment:
completed_assessment is misspelled throughout... if this changes, other references in the code will need to change